modules/ossm-rn-known-issues.adoc

```diff
@@ -21,21 +21,34 @@ These limitations exist in {SMProductName}:
 
 * The first time you access related services such as {JaegerShortName} and Grafana, from the Kiali console, you must accept the certificate and re-authenticate using your {product-title} login credentials. This happens due to an issue with how the framework displays embedded pages in the console.
 
-* The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
+* The Bookinfo sample application cannot be installed on IBM Z and IBM Power.
 
-* WebAssembly extensions are not supported on IBM Z and IBM Power Systems.
+* WebAssembly extensions are not supported on IBM Z and IBM Power.
 
+* LuaJIT is not supported on IBM Power.
 
 [id="ossm-rn-known-issues-ossm_{context}"]
 == {SMProductShortName} known issues
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-1668[OSSM-1668]
+`jwksResolverCA` field is missing in `SMCP`.
```

Comment:
This needs more specific details.  See the template for Release Notes (what user action might make this problem appear?  Has a cause been identified?  Is there a workaround?)

Reply:
The known issue OSSM-1668 is related to https://issues.redhat.com/browse/OSSMDOC-614
You can add a note about that issue was fixed in OSSM 2.1.x SMCP and the same configuration spec is not working in OSSM 2.2.0 SMCP. Our fix target release will be OSSM 2.2.1

Reply:
@rh-tokeefe You can add user action about that known issue.
In OSSM 2.2.0 SMCP, the jwksResolverCA spec field is not supported and a user need to use the previous Tech preview field for enabling additional JWKS CA certificates.

Reply:
@yxun I've updated this to the following:

If you upgrade from Service Mesh operator 2.1.3 to Service Mesh operator 2.2, then the jwksResolverCA field is not supported. You must use the techPreview jwksResolverExtraRootCA field to enable additional JWKS CA certificates.

The change should now appear in the preview build.

cc: @longmuir and @JStickler